src/plone/restapi/tests/test_services_upgrade.py

```diff
@@ -0,0 +1,72 @@
+from plone.app.testing import SITE_OWNER_NAME
+from plone.app.testing import SITE_OWNER_PASSWORD
+from plone.restapi.testing import PLONE_RESTAPI_DX_FUNCTIONAL_TESTING
+from plone.restapi.testing import RelativeSession
+from Products.CMFCore.utils import getToolByName
+from Products.CMFPlone.factory import _DEFAULT_PROFILE
+
+import transaction
+import unittest
+
+
+# Python 3 is only supported on 5.2+.
+# This means you can not upgrade from 5.1 or earlier.
+START_PROFILE = "5200"
```

Comment:
I'd call it START_VERSION

Reply:
This was copied verbatim from MigrationTool tests. (But I agree with you)